Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.12.4-alpine
+
+COPY . /tmp/src
+RUN pip install --no-cache-dir /tmp/src && \
```

Comment:
For the image that we upload to the registry I would rather install the package from PyPI here.